packages/react-components/react-tag-picker/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ This log was last generated on Mon, 06 May 2024 12:55:01 GMT and should not be m
 
 ## [0.5.0](https://github.com/microsoft/fluentui/tree/@fluentui/react-tag-picker-preview_v0.5.0)
 
-Mon, 06 May 2024 12:55:01 GMT 
+Mon, 06 May 2024 12:55:01 GMT
```

Comment:
Just curious, since this file is autogenerated why should there be differences in whitespace?

Do you know @Hotell ?